crypto/evp_extra/evp_tests.txt

```diff
@@ -1655,6 +1655,16 @@ Digest = SHA256
 Input = "Hello world"
 Output = 301894798b49d6ec55d32dcc74314f04230591a515781f3eb4492f5324b56046836c4bc3e25942af341e88558cb4c3814a849207575d343189147989b16e296b5138dbbc717116dc416f201dfa35943d15060493953cda1f04a13ff89845cf7fd69e1a78d5d38522a77bb234e5d0ba2ae17ada6e22fdae27a4052fdb8ac267507dfe06ed7a865e61a52b530bbbf65c7caa89739613df10ae3b0e62ff6831ee0770086aad39c329462aede9f1b29a501bc3d09e0fe4034aa5d6831d44491d508111d88a1d7ba50cee5ef7e701b3a589adc09a752a974a6805956f4a1a0582f66309a1e02e9fb6b10d2c820fe98bb2eb04f435bc8a649cc9ab6c5a4c03e83800d1
 
+SignMessage = RSA-2048
+Digest = SHA512/256
+Input = "Hello world"
+Output = 8beb180f07b45b17af94c49badf8a7a630e89ffdaade1c3e2d0845ada27188a5d2d991926650773edfb5d36de5237abb82a1009e5dba5c7080b8fe19bd08227273420b73c060fb2efad53e0936b17cd341c7c02e588c7b0a3138c1205a7d46f8c5648267104aae1b285b390d90378aad9431499cf24feec2243ff668c98481940b08a10b63da8e7423f7e7e307485b6e7a6c73a5b68d70a36ab7cc5cb634773b6f1f8651932ea80c714501853e9fe1f2eff9b67fe03631e730345e02d14fa1da5fcbe52299c8b614fe5c4d17fdc9bd4f49b659154a95914fee33bce64c0cc4ba9aaf0809680b75cd891120025597a5b47e44cb3c669118591927133202768cd7
+
+VerifyMessage = RSA-2048
+Digest = SHA512/256
+Input = "Hello world"
+Output = 8beb180f07b45b17af94c49badf8a7a630e89ffdaade1c3e2d0845ada27188a5d2d991926650773edfb5d36de5237abb82a1009e5dba5c7080b8fe19bd08227273420b73c060fb2efad53e0936b17cd341c7c02e588c7b0a3138c1205a7d46f8c5648267104aae1b285b390d90378aad9431499cf24feec2243ff668c98481940b08a10b63da8e7423f7e7e307485b6e7a6c73a5b68d70a36ab7cc5cb634773b6f1f8651932ea80c714501853e9fe1f2eff9b67fe03631e730345e02d14fa1da5fcbe52299c8b614fe5c4d17fdc9bd4f49b659154a95914fee33bce64c0cc4ba9aaf0809680b75cd891120025597a5b47e44cb3c669118591927133202768cd7
```

Comment:
Is this duplicated on purpose?

Reply:
Discussed on slack but leaving this here for posterity:

Not a duplicate, the test above it is `SignMessage` while this one is `VerifyMessage`. They take the same parameter otherwise but just run `EVP_DigestSign()` vs. `EVP_DigestVerify()`.